### ISSUE TITLE
Latest Moov client throws TypeError when constructed

### ISSUE THREAD

**Issue by codyjk** [moovfinancial/moov-node#80]
Version: 1.2.0

We are getting a `TypeError` when constructing a Moov client instance in our application code. This is a breaking change and broke our integration with Moov. Luckily we found a workaround before this was in use in our production environment.

Our code constructs the client like this:

```ts
      this.#client = new Moov({
        accountID: secrets.accountId,
        publicKey: secrets.publicKey,
        secretKey: secrets.privateKey,
        domain: secrets.domain,
      });
```

According to the type signature and the [documentation](https://docs.moov.io/moovjs/authentication/), this is the correct way to construct the client.

However, it doesn't work, and it throws a `TypeError`:

```
TypeError: Expected value which is predicate returns truthy for any value, received values of types undefined.
  ...
  at new Moov (/Users/cjk/backend/node_modules/@moovio/node/dist/cjs/moov.js:226:34)
  at Function.got.extend (/Users/cjk/backend/node_modules/got-cjs/dist/source/create.js:111:25)
  at Options.merge (/Users/cjk/backend/node_modules/got-cjs/dist/source/core/options.js:441:27)
  at Options.set prefixUrl [as prefixUrl] (/Users/cjk/backend/node_modules/got-cjs/dist/source/core/options.js:610:21)
  at Object.any (/Users/cjk/backend/node_modules/@sindresorhus/is/dist/index.js:402:16)
  at assertType (/Users/cjk/backend/node_modules/@sindresorhus/is/dist/index.js:303:15)
```

By digging through the [code](https://github.com/moovfinancial/moov-node/blame/main/lib/moov.js#L218-L242) we were able to fix the issue by supplying `apiRoot`. The code works but does not typecheck correctly.

```ts
      this.#client = new Moov({
        accountID: secrets.accountId,
        publicKey: secrets.publicKey,
        secretKey: secrets.privateKey,
        domain: secrets.domain,
        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
        // @ts-ignore
        apiRoot: 'https://api.moov.io',
      });
```

With this workaround we were able to restore the integration with Moov. Please advise on if there is another way to fix this.

**Comment by sethlivingston** [comment]
Hi, @codyjk. Posting a fix for this immediately. Thank you.

**Comment by sethlivingston** [comment]
This is available in v1.2.1. Let me know if you encounter any more problems!
